.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "useTabs": false,
```

Comment:
The other day I read this, and it open my eyes to tabs vs spaces: https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/